estimation_tracker/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for estimation_tracker project.
+
+Generated by 'django-admin startproject' using Django 4.0.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
Is that generated directly when creating a django project with Django 4?

Reply:
yes